src/loaders/lottie/tvgLottieProperty.h

```diff
@@ -238,6 +239,88 @@ static void _copy(PathSet& pathset, Array<PathCommand>& outCmds)
 }
 
 
+static void _roundCorner(Array<PathCommand>& cmds, Array<Point>& pts, const Point& prev, const Point& curr, const Point& next, float roundness, bool continuity, Point& distPrev)
+{
+    auto lenPrev = mathLength(prev - curr);
```

Comment:
It looks, we can make it optimal by avoiding sqrt() call.

Reply:
that was the intention while introducing continuity, but I've mixed up something. now fixed

Reply:
@mgrudzinska If you think this makes sense, feel free to change it; otherwise, you can keep it as it is.